democracylab/emails.py

```diff
@@ -292,3 +292,14 @@ def _get_co_owner_emails(project):
 
 def _get_account_from_email(email_acct):
     return email_acct['from_name'] if email_acct is not None else 'DemocracyLab'
+
+def contact_democracylab_email(fname, lname, emailaddr, body):
+    subject = '{} {} would like to contact DemocracyLab'.format(fname, lname)
+    email_msg = EmailMessage(
+        subject=subject,
+        body=body,
+        from_email=[emailaddr],
```

Comment:
```suggestion
        from_email=_get_account_from_email(settings.EMAIL_SUPPORT_ACCT),
```
from_email is an account object, it's not enough to specify just an email address.